tests/scenarios/FAILING_IN_GHA_github_test_13/cmd1

```diff
@@ -0,0 +1,7 @@
+source: ubuntu-1-deployment
```

Comment:
FAILING_IN_GHA? do you know why it fails?

Reply:
@nam-jaehyun, I and @daemon1024 suspect that it fails just like `FAILING_IN_GHA_github_test_09` because ping in GH actions doesn't use raw sockets and thus it is not audited. Please correct if you think this might not be accurate.

Reply:
in that case, we can use "arping -c 1 127.0.0.1" to create raw sockets

Reply:
@nam-jaehyun updated.
Sorry for taking so long.. I messed up my env while trying out CRIs :sweat_smile:

Reply:
Also, if this works, should we also update the other commented out tests i.e `test_09` and `test_10`? :thinking:

Reply:
@DelusionalOptimist we can update those test scenarios in another PR.

Reply:
Yes we should certainly, if this works.